packages/playground/src/utils/date.ts

```diff
@@ -3,3 +3,27 @@ import moment from "moment";
 export default function toHumanDate(timeInSeconds: number): string {
   return moment(timeInSeconds * 1000).format("DD MMM YYYY, hh:mm A");
 }
+
+/**
+ * Checks if the provided date is within the last 30 days.
+ *
+ * @param {Date} date - The date to check.
+ * @returns {boolean} True if the date is within the last 30 days, false otherwise.
+ *
+ * @description
+ * This function calculates the difference in days between the provided date and the current date.
+ * It then checks if this difference is less than or equal to 30 days.
+ *
+ * @example
+ * const date = new Date('2022-01-01');
+ * console.log(thirtyDaysInMilliseconds(date)); // Output: true or false depending on the current date
+ */
+
+export function thirtyDaysInMilliseconds(date: Date): boolean {
```

Comment:
the name is wrong for this one, this one returns if the difference between two dates (one of them is fixed as the current one and the other one is the release date) so should be

isDifferenceBetweenDates30Days(date1, date2) and when you use it pass date1 as the current and date2 gets the value of the release date